src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.split(';');
+  const stylesObject = {};
+
+  stylesArray.forEach((style) => {
+    if (style.trim() !== '') {
+      const [key, value] = style.split(':').map((item) => item.trim());
+
+      stylesObject[key] = value.includes(' ') ? `${value}` : value;
```

Comment:
```suggestion
      stylesObject[key] = value;
```
looks redundant for me